env.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
It seems this should be a temporary change and should be removed in the final implementation.

And I am not sure about merging it to the master branch directly. Probably, we should use some separate development brunch for pre-stage tasing and further releasing.